app/src/README.md

```diff
@@ -1,17 +1,29 @@
-Node/Grunt modules for backend’s frontend workflow.
+# Bolts backend’s frontend workflow using grunt
```

Comment:
Oh, I wasn't in the mood of writing documentation yesterday. I was hoping someone with a mother tongue would be … ;-)